src/scene-entry-manager.js

```diff
@@ -581,7 +581,7 @@ export default class SceneEntryManager {
 
     const getAudio = () => {
       audioEl.loop = true;
-      audioEl.muted = true;
+      audioEl.muted = false;
```

Comment:
This makes the bots unmuted when they enter yeah? I wonder why that was switched to true.

Reply:
I'm not sure about that, that line goes back quite a long time. I guess maybe before new UI we were un-muting that? The fact is that if it's that without that it doesn't work.